source/scss/_pf-component-view-selector.scss

```diff
@@ -0,0 +1,15 @@
+//
+// View Selector component
+//
+
+.pf-view-selector__btn-link {
+  color: $color-pf-black-900;
+  font-size: $font-size-relative-base;
+  padding: 0;
```

Comment:
Aren't they too small on a small device? We can merge this and add that todo to the follow up story. @srambach what do you think?